src/model_io/urdf/src/URDFModelExport.cpp

```diff
@@ -106,10 +106,120 @@ bool exportInertial(const SpatialInertia &inertia, xmlNodePtr parent_element)
     return ok;
 }
 
+
+
+/*
+ * Add a <visual> or <collision> URDF element to the specified parent element for the specified solid shapes.
+ *
```

Comment:
Maybe we can also add this little detail,

"URDF considers <box>, <cylinder>, <sphere> and <mesh> as shapes."